tests/python/frontend/tflite/test_forward.py

```diff
@@ -723,6 +723,14 @@ def _test_greater(data):
     """ One iteration of greater """
     return _test_elemwise(math_ops.greater, data)
 
+#######################################################################
+# Squared_difference
+# -------
```

Comment:
Align - the same length of squared_difference

Reply:
fixed